client/components/Header/Logo.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import logo from '../../images/mclogo.svg';
```

Comment:
I just read a great blog post by a super cool dude about this. His name is Grg...hey, wait a minute...